src/_pytest/config/findpaths.py

```diff
@@ -113,7 +114,10 @@ def locate_config(
     return None, None, {}
 
 
-def get_common_ancestor(paths: Iterable[Path]) -> Path:
+def get_common_ancestor(
+    invocation_dir: Path,
+    paths: Iterable[Path],
+) -> Path:
```

Comment:
```suggestion
def get_common_ancestor(
    paths: Iterable[Path],
    fallback_default: Path,
) -> Path:
```
i want to be a little nit-picky about what to name the arguments as some of the functions are not that pytest specific

Reply:
I see where you are coming from, but on the other hand those are private/internal functions and using `invocation_dir` makes it crystal clear and unambiguous what the parameter should be, so there is that to consider.

Reply:
I agree with @nicoddemus, it's pretty useful at a glance to see that this is the invocation dir. Of course if the function will ever be used more generally then we should change the name.